src/pages/Popup/components/Settings.jsx

```diff
@@ -59,6 +64,9 @@ class Settings extends Component {
   handleSwiChange = async (event) => {
     await this.setState({ selectedSwi: event.target.checked });
     this.setState({ buttonDisabled: false });
+    this.setState({
```

Comment:
Add the background state in the above `this.setState` function itself. No need to write it again

Reply:
Updated.